internal/v1/blueprint_rules.go

```diff
@@ -0,0 +1,334 @@
+package v1
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+
+	"github.com/labstack/echo/v4"
+	"github.com/osbuild/images/pkg/customizations/fsnode"
+)
+
+var (
+	blueprintNameRegex         = regexp.MustCompile(`\S+`)
+	blueprintInvalidNameDetail = "The blueprint name must contain at least two characters."
```

Comment:
Errors in go should not start with capital letter, when you join them it looks weird: `some error: Something happened`